app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -34,5 +32,37 @@ def redirect_with_flash
       flash[:danger] = 'Please Log In with your organization\'s Google Account.'
       redirect_to new_user_session_path
     end
+
+    def create_organization_and_track(user, company_domain)
+      user.organization = Organization.create(
```

Comment:
i tried this way....
`2.3.1 :011 > user.organization.create
NoMethodError: undefined method `create' for nil:NilClass`